app/src/main/res/layout/activity_change_contact.xml

```diff
@@ -6,7 +6,8 @@
     android:layout_height="match_parent"
     android:fitsSystemWindows="true"
     android:orientation="vertical"
-    tools:context="com.nervousfish.nervousfish.activities.ContactActivity">
+    tools:context="com.nervousfish.nervousfish.activities.ContactActivity"
+    tools:ignore="LabelFor">
```

Comment:
Lint says that every Edittext should have another element pointing to it with Labelfor, but here that is not suited.